stations.csv

```diff
@@ -70310,3 +70310,57 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
+74487;Campdevànol;campdevanol;7177301;;42.221558;2.165877;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74487;
+74488;Planoles;planoles;7177304;;42.313559;2.103078;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74488;
+74489;Toses;toses;7177305;;42.324189;2.029728;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74489;
+74490;Urtx Alp;urtx-alp;7177307;;42.3761666;1.897009;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74490;
+74491;Montcada I Reixac;montcada-i-reixac;7179005;;41.483833;2.188193;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74491;
+74492;Mollet Sant Fost;mollet-sant-fost;7179006;;41.5325016;2.2134974;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74492;
+74493;Montmelo;montmelo;7179007;;41.5501609;2.2450896;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74493;
+74494;La Llagosta;la-llagosta;7179011;;41.5104164;2.1993799;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74494;
+74495;Palautordera;palautordera;7179103;;41.671758;2.454429;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74495;
+74496;Sant Adrià De Besòs;sant-adria-de-besos;7179403;;41.424665;2.230761;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74496;
+74497;Montgat Nord;montgat-nord;7179406;;41.468941;2.286615;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74497;
+74498;El Masnou;el-masnou;7179407;;41.477702;2.314362;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74498;
+74499;Ocata (Apd);ocata-apd;7179408;;41.4789679;2.3165779;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74499;
+74500;Premiá De Mar;premia-de-mar;7179409;;41.4875728;2.354368;;ES;Europe/Madrid;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74500;
```

Comment:
It's the wrong accent I think: `Premià de Mar`

Reply:
This should be fixed